chrome-launcher.ts

```diff
@@ -79,7 +79,7 @@ export async function launch(opts: Options = {}): Promise<LaunchedChrome> {
     return instance.kill();
   };
 
-  return {pid: instance.pid!, port: instance.port!, kill};
+  return {pid: instance.pid!, port: instance.port!, kill, proc: instance.chrome};
```

Comment:
is there a particular reason we're exposing under `.proc` instead of `.process` or `.processInstance` or something? 

I'm totally fine if there's precedent and we're matching what several other node libraries expose, but if not, I'd prefer clarity

Reply:
I was worried I'd  get some nitpicks on that, I don't care what we call it. We can call it `chromeProcess` if you want?

Reply:
Do you know why the build failed?

Reply:
`chromeProcess` sgtm 👍 

and yes, `.chrome` is marked as private in typescript you'll need to move the declaration in `class Launcher` to be with the other public properties

Reply:
ok cool, np, I just updated the PR, hopefully it will take thx